python/ray/serve/_private/proxy_state.py

```diff
@@ -318,15 +319,19 @@ def __init__(
         actor_name: str,
         node_id: str,
         node_ip: str,
+        proxy_restart_count: int = 0,
+        time_getter: ModuleType = time,
```

Comment:
`ModuleType` is a bit funky.

How about: `Callable[[], float]` instead? (function with no args that returns a float)

Then you'd pass `time.time` by default

Reply:
or else, maybe best case, define a `Time` abstract base class that matches the relevant `time` module functions (and have `FakeTime` implement it)

Reply:
Sg! Will create an abstract class!